market_maker_keeper/uniswapv2_util.py

```diff
@@ -0,0 +1,287 @@
+#
```

Comment:
support for UniswapV2 contract is better suitable to stay in pyexchange project . Abi file can be a separate one as https://github.com/makerdao/pyexchange/tree/master/pyexchange/abi

Reply:
This has been removed